.github/workflows/reusable-deployment.yml

```diff
@@ -376,6 +382,20 @@ jobs:
         uses: ./.github/actions/cache-unplugged
         with:
           environment-name: ${{ inputs.environment-name }}
+
+      - name: Deploy CRN PR
```

Comment:
I would be tempted to merge the two. For conditions it looks like we want to skip if it's a branch but the label isn't set. As for the config I think where it varies you could probably have them passed into this reusable workflow

Reply:
The problem is that the PR CI secret is different and needs to be to support both flows. Its really a different case. I tried to make it one job initially.